airbyte-integrations/connectors/source-bing-ads/unit_tests/test_reports.py

```diff
@@ -38,6 +38,14 @@ def test_get_column_value():
     assert test_report.get_column_value(record, "Spend") == 1.203
 
 
+def test_get_updated_state_init_state():
```

Comment:
This unit test makes sure that the state is correctly initiated for a given stream.

Nothing related to the fix, but I took the opportunity to add it.